plugins/by-name/scrollview/default.nix

```diff
@@ -0,0 +1,444 @@
+          [
+            "left"
+            "right"
+          ]
+          ''
+            The sign overflow direction (to avoid overlapping the scrollbar or other signs).
+          '';
+
+      signs_show_in_folds = defaultNullOpts.mkBool false ''
+        Whether signs on lines within hidden folds should be shown.
+        Sign groups can override this setting (e.g., the built-in cursor sign group does).
+      '';
+
+      changelist_previous_priority = mkPriorityOption "the previous item changelist sign" 15;
+
+      changelist_previous_symbol = mkNullOrStr ''
+        The symbol for the previous item changelist sign.
+
+        Defaults to an upwards arrow with tip leftwards.
+      '';
+
+      changelist_current_priority = mkPriorityOption "the current changelist sign" 10;
+
+      changelist_current_symbol = defaultNullOpts.mkStr "@" ''
+        The symbol for the current item changelist sign.
+      '';
+
+      changelist_next_priority = mkPriorityOption "the next item changelist sign" 5;
+
+      changelist_next_symbol = mkNullOrStr ''
+        The symbol for the next item changelist sign.
+
+        Defaults to a downwards arrow with tip rightwards.
+      '';
+
+      conflicts_bottom_priority = mkPriorityOption "conflict bottom signs" 80;
+
+      conflicts_bottom_symbol = defaultNullOpts.mkNullable symbolType ">" ''
+        The symbol for conflict bottom signs.
+
+        ${symbolCommonDesc}
+      '';
+
+      conflicts_middle_priority = mkPriorityOption "conflict middle signs" 75;
+
+      conflicts_middle_symbol = defaultNullOpts.mkNullable symbolType "=" ''
+        The symbol for conflict middle signs.
+
+        ${symbolCommonDesc}
+      '';
+
+      conflicts_top_priority = mkPriorityOption "conflict top signs" 70;
+
+      conflicts_top_symbol = defaultNullOpts.mkNullable symbolType "<" ''
+        The symbol for conflict top signs.
+
+        ${symbolCommonDesc}
+      '';
+
+      cursor_priority = mkPriorityOption "cursor signs" 0;
+
+      cursor_symbol = mkNullOrStr ''
+        The symbol for cursor signs.
+
+        Defaults to a small square, resembling a block cursor.
+      '';
+
+      diagnostics_error_priority = mkPriorityOption "cursor diagnostic error signs" 60;
+
+      diagnostics_error_symbol = mkNullOrOption symbolType ''
+        The symbol for diagnostic error signs.
+
+        Defaults to the trimmed sign text for `DiagnosticSignError` if defined, or `"E"` otherwise.
+
+        ${symbolCommonDesc}
+      '';
+
+      diagnostics_hint_priority = mkPriorityOption "the diagnostic hint signs" 30;
+
+      diagnostics_hint_symbol = mkNullOrOption symbolType ''
+        The symbol for diagnostic hint signs.
+
+        Defaults to the trimmed sign text for `DiagnosticSignHint` if defined, or `"H"` otherwise.
+
+        ${symbolCommonDesc}
+      '';
+
+      diagnostics_info_priority = mkPriorityOption "the diagnostic info signs" 40;
+
+      diagnostics_info_symbol = mkNullOrOption symbolType ''
```

Comment:
IIRC, `mkNullOrOption` is one of the weird special cases that don't inject `maybeRaw` support? If so there may be a `mkNullOrOptionWithRaw` variant or you could manually add `rawLua` or `maybeRaw` support.

Reply:
You are right, I have fixed that and adjusted the test too.